fetch.bs

```diff
@@ -4039,17 +4039,7 @@ fetch("/music/pk/altes-kamuffel.flac")
 
 <h3 id=headers-class>Headers class</h3>
 
-<pre data-no-idl class=idl>typedef (Headers or sequence&lt;sequence&lt;ByteString>> or OpenEndedDictionary&lt;ByteString>) <dfn id=headersinit typedef export>HeadersInit</dfn>;</pre>
-
-<div class=XXX>
- <p>OpenEndedDictionary&lt;T> is a future IDL construct. Expect it to be used as such:
-
- <pre>
-var meta = { "Content-Type": "text/xml", "Breaking-Bad": "&lt;3" }
-new Headers(meta)</pre>
```

Comment:
I would prefer to keep this example. Change the class on the `<div>` to `example` and change the opening paragraph to something like: `Constructing a {{Headers}} object is fairly straightforward:`